pootle/apps/pootle_store/unit/results.py

```diff
@@ -88,7 +73,6 @@ def data(self):
                     'project_style': unit.project_style}
             units_list.append(
                 {'id': unit.id,
-                 'url': unit.translate_url,
```

Comment:
Notice how this [is not dead code](https://github.com/translate/pootle/blob/master/pootle/templates/editor/units/xhr_view.html#L2), and moreover the change breaks opening editing rows in new tabs.